Multisig/UI/ClaimToken/ClaimingAmountViewController/AllocationBoxCell.swift

```diff
@@ -0,0 +1,227 @@
+//
+//  AllocationBoxCell.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 01.09.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class AllocationBoxCell: UITableViewCell {
```

Comment:
Isn't it better to have the allocation box contains two sections as before ? one for available and the other for future claiming?

Reply:
every allocation box is essentially the same but with different configurations, so I implemented one allocation box and use it for 4 different configurations (guardian / user, dark / light) - which is set by the "style" property